features/cli/oc_set_build_hook.feature

```diff
@@ -36,18 +36,4 @@ Feature: oc_set_build_hook
     When I run the :start_build client command with:
       | buildconfig | rails-postgresql-example |
     Then the step should succeed
-    Given the "rails-postgresql-example-3" build completes
-    When I run the :set_build_hook client command with:
```

Comment:
If the set hook feature is removed from 4.x?

Reply:
No, just set hook commond did not support with args options in 4.x. so remove the part. there's a bug: https://bugzilla.redhat.com/show_bug.cgi?id=1670000

Reply:
bug shows verified. Do we still need to remove those?

Reply:
Hi @akostadinov you can see https://github.com/openshift/origin/pull/21880/files#diff-77d633c62fb7fa312b6e7edff1743c7cL2942 in pr https://github.com/openshift/origin/pull/21389/files, codes already remove it, so scripts removed it too.